.github/workflows/azure-dev.yml

```diff
@@ -105,6 +105,17 @@ jobs:
         with:
           node-version: 18
 
+      - name: Run Microsoft Security DevOps Analysis
+        uses: microsoft/security-devops-action@v1
+        id: msdo
+        with:
+          tools: templateanalyzer
+
+      - name: Upload results to Security tab
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: ${{ steps.msdo.outputs.sarifFile }}
```

Comment:
@tonybaloney is moving to using PSRules now, as its less flaky then templateanalyzer

Reply:
Ok, maybe I would like for us to have some benchmarks. If we're recommending PSRules over msdo, we should have some metrics. Does @tonybaloney have any such data?

Reply:
I removed the actions for now and the validation will throw a warning instead of an error, while we decide what tool we promote as standard.